homework.md

```diff
@@ -86,19 +87,19 @@ You send the changes of your local repository to Github
 
 What is Github?
 
-A place for us to store and share our code.
+GitHub is one website where you can use Git.
```

Comment:
I suggest to double check this answer and make it a bit more understanable/detailed